src/components/Marker/Markers.js

```diff
@@ -1,14 +1,13 @@
-import React, { Component } from "react";
+import React, { PureComponent } from "react";
 import { Marker as LeafletMarker } from "react-leaflet";
 import ReactDOMServer from "react-dom/server";
 import {
   createIcon,
   defaultMarker,
   getChildrenProperties
 } from "./Utilities/IconUtils";
-import areEqual from "fbjs/lib/areEqual";
 
-class Marker extends Component {
+class Marker extends PureComponent {
```

Comment:
PureComponent intrinsically does a shallow comparison, considering this package handles nested components we need to go with areEqual method of fbjs as it handles deep comparison and need to override the shouldComponentUpdate lifeCycle explicitly!!

Reply:
I know about this specification. But for current case that's not needed